client/src/App.js

```diff
@@ -1,17 +1,22 @@
-import React from "react";
```

Comment:
It looks like you're using a different code formatter. These kinds of changes clutter a PR.

Reply:
Ok, I see, I have 'Prettier NOW'. will go with 'Prettier code formatter'